relay-general/src/protocol/contexts.rs

```diff
@@ -524,7 +524,8 @@ impl fmt::Display for SpanStatus {
             SpanStatus::Unimplemented => write!(f, "unimplemented"),
             SpanStatus::Unavailable => write!(f, "unavailable"),
             SpanStatus::InternalError => write!(f, "internal_error"),
-            SpanStatus::UnknownError => write!(f, "unknown_error"),
+            // TODO: Switch this to "unknown" once snuba is updated
+            SpanStatus::Unknown => write!(f, "unknown_error"),
```

Comment:
Is APM enabled on single tenant already?

Reply:
Fair point, indeed we do ingest that data. Let's just wait two weeks then I guess